Fix default page colors

Pinpickle opened the pull request:
Before:

![image](https://cloud.githubusercontent.com/assets/3238878/19596412/66e46030-9786-11e6-9858-8ab71d0c8717.png)

After:

![image](https://cloud.githubusercontent.com/assets/3238878/19596422/6e02080e-9786-11e6-8239-67df053eb1a3.png)

These pages aren't works of art but they're more for utility and aren't often visited. Also, our /pay page looks a little more in line now:

![image](https://cloud.githubusercontent.com/assets/3238878/19596461/a26ccbb0-9786-11e6-837c-4bc47eb6a3fe.png)

@varkor could you review?

varkor commented:
That's a lot better. Couple of questions: we could want to override `::selection` for `dark-background()`, as the grey doesn't blend very well with the red, though this might be overly picky.
Also, do you think purple is the most appropriate secondary colour? Do you think a darker grey (or desaturated red) would work?

Pinpickle commented:
Purple was kind of arbitrary, it's not really used anywhere that depends on it greatly. I was just trying to change it from white and tear out the dependency on it for the text colour.

I would vote against desaturated red though, as that is a variation on the primary colour that we can create that with functions.

Pinpickle commented:
@varkor I've switched the secondary color to (another) arbitrary color, and removed the selection styling. While making our own is really nice, there are so many special cases we need to account for.

For example, if we have `dark-background` automatically change the selection, what happens when we have a light background element inside it? We'd have to remember to redefine the selection in these cases as well.

varkor commented:
If we just go for the red as the selection colour, on every background (light or dark) apart from red, it will show up. Then we can just redefine the colour for red background elements (it should be easy enough to specify for these cases, which will probably inherit from a common class anyway. But I'm happy to leave that for now — it'll probably be easier to do at the end (if we wanted) anyway.

So that said — I think we can merge this now!
